wicket/src/cli/inventory.rs

```diff
@@ -0,0 +1,123 @@
+                // We don't use the /bootstrap-sleds endpoint, because that
+                // gets all sleds visible on the bootstrap network. We want
+                // something different here.
+                // - We want the status of only sleds we've configured wicket
+                //   to use for setup. /bootstrap-sleds will give us sleds
+                //   we don't want
+                // - We want the status even if they aren't visible on the
+                //   bootstrap network yet.
+                //
+                // In other words, we want the sled information displayed at the
+                // bottom of the rack setup screen in the TUI.
+                let conf = client
+                    .get_rss_config()
+                    .await
+                    .context("failed to get rss config")?;
+
+                let bootstrap_sleds = &conf.insensitive.bootstrap_sleds;
+                let sled_data: Vec<ConfiguredBootstrapSledData> =
+                    bootstrap_sleds
+                        .iter()
+                        .map(|desc| {
+                            let cubby = desc.id.slot;
+
+                            let identifier = match &desc.baseboard {
+                                Baseboard::Gimlet { identifier, .. } => {
+                                    identifier.clone()
+                                }
+                                Baseboard::Pc { identifier, .. } => {
+                                    identifier.clone()
+                                }
+                                Baseboard::Unknown => "unknown".to_string(),
+                            };
+
+                            let address = desc.bootstrap_ip;
+
+                            ConfiguredBootstrapSledData {
+                                cubby,
+                                identifier,
+                                address,
+                            }
+                        })
+                        .collect();
+
+                if json {
+                    let json_str = serde_json::to_string(&sled_data)
+                        .context("serializing sled data")?;
+                    println!("{}", json_str);
+                } else {
+                    for sled in &sled_data {
+                        print_bootstrap_sled_data(sled);
+                    }
+                }
+
+                Ok(())
+            }
+        }
+    }
+}
+
+#[derive(Serialize)]
+struct ConfiguredBootstrapSledData {
+    cubby: u32,
+    identifier: String,
+    address: Option<Ipv6Addr>,
+}
+
+fn print_bootstrap_sled_data(data: &ConfiguredBootstrapSledData) {
+    let ConfiguredBootstrapSledData { cubby, identifier, address } = data;
+
+    // Print status indicator
+    let status = match address {
+        None => format!("{}", '⚠'.red()),
+        Some(_) => format!("{}", '✔'.green()),
+    };
+
+    let addr_fmt = match address {
+        None => "not_available".to_string(),
```

Comment:
Can this be `(not available)`?

Reply:
I'll admit I made it not_available to make it one field in awk even if you are using all whitespace as your field separator. but i think that's probably a bad reason given we have the json output *and* the data is tab-separated so yeah I'll change that